services/block.go

```diff
@@ -164,7 +165,23 @@ func (s *BlockAPIService) Block(
 		// if err != nil {
 		//	 return nil, err
 		// }
-		parsedTraces, discoveredAddresses, parseError = s.p.ParseTransactions(states.Trace, tipSet, nil) // TODO: fill with ethLogs
+		tracesBytes, marshalErr := json.Marshal(states.Trace)
+		if marshalErr != nil {
+			return nil, rosetta.BuildError(rosetta.ErrUnableToGetTrace, marshalErr, true)
```

Comment:
I don't know if `rosetta.ErrUnableToGetTrace` is the correct error. If not, which one?

Reply:
that error should be returned after GetStateCompute, above